savant/meta/bbox.py

```diff
@@ -48,10 +50,48 @@ def left(self, value: float):
         left corner."""
         self.x_center = value + 0.5 * self.width
 
+    def scale(self, scale_x: float, scale_y: float):
+        """Scales BBox.
+
+        :param scale_x:
```

Comment:
Please add a description of the parameters. Something like "The scaling factor to apply along the x-axis"

Reply:
fixed